modules/social_features/social_event/social_event.module

```diff
@@ -858,7 +926,9 @@ function social_event_node_update(EntityInterface $entity) {
       foreach ($enrollments->getAllEventEnrollments($entity->id(), TRUE) as $enrollment) {
         // Check if the field is not empty before getting the value,
         // Since (int) will otherwise return 0 and give false positives.
-        if (!$enrollment->get('field_request_or_invite_status')->isEmpty() && (int) $enrollment->get('field_request_or_invite_status')->getString() === EventEnrollmentInterface::REQUEST_PENDING) {
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.